### PR TITLE
Timer design

### PR DESCRIPTION
**Pull request by taltaub22**

### CODE REVIEW

**Comment by alanggreen** [comment]
What is a reveal?

File: src/js/components/container/Timer.jsx [hunk 1]
```diff
@@ -44,10 +44,16 @@ class Timer extends Component {
...
-    open={this.state.running} closeStyle={{'display': 'none'}}>
-      <div className="text-center h1">{this.state.time}</div>
+  render () {
+    const overlayStyle = {
+      'backgroundColor': 'rgba(0,0,0,0)'
+    }
+
+    return <Modal isModal size="tiny"
+                  overlayStyle={overlayStyle}
+                  style="border: 5px black solid;"
```
[the remark this answers is not in the document]
Comment (taltaub22) [reply src/js/components/container/Timer.jsx]: That's not even working. Its there for no real reason

File: src/js/components/container/Timer.jsx [hunk 2]
```diff
@@ -44,10 +44,23 @@ class Timer extends Component {
     })
   }
 
-  render() {
-  	return <Modal isModal size="tiny"
-    open={this.state.running} closeStyle={{'display': 'none'}}>
-      <div className="text-center h1">{this.state.time}</div>
+  render () {
+    const overlayStyle = {
+      'backgroundColor': 'rgba(0,0,0,0)'
```
Comment (idanstark42) [inline src/js/components/container/Timer.jsx]: Why is this still here and not in the matching CSS file?

Comment (taltaub22) [reply src/js/components/container/Timer.jsx]: I could nit add class/id to the modal. The only way I found is to pass Jsons to the modal it self.
https://www.npmjs.com/package/react-foundation-modal#props

Comment (idanstark42) [reply src/js/components/container/Timer.jsx]: Fine. It'll be fine when we user semantic

**Pull request merged**